contracts/mainnet/connectors/euler/main.sol

```diff
@@ -376,105 +376,37 @@ abstract contract Euler is Helpers {
 	}
 
 	/**
-	 * @dev Approve debt.
-	 * @notice Approves receiver to take debt.
-	 * @param subAccountId Subaccount number
-	 * @param debtReceiver Address of receiver
-	 * @param token The address of the token to mint.(For ETH: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE)
-	 * @param amt The amount of the token to mint.
-	 * @param getId ID to retrieve amt.
+	 * @dev Approve Spender's debt.
+	 * @notice Approve sender to send debt.
+	 * @param subAccountId Subaccount id of receiver
+	 * @param debtSender Address of sender
+	 * @param token The address of the token.(For ETH: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE)
+	 * @param amt The amount of the token.
 	 * @param setId ID stores the amount of tokens deposited.
 	 */
-	function approveDebt(
+	function approveSpenderDebt(
 		uint256 subAccountId,
-		address debtReceiver,
+		address debtSender,
 		address token,
 		uint256 amt,
-		uint256 getId,
 		uint256 setId
 	)
 		external
 		payable
 		returns (string memory _eventName, bytes memory _eventParam)
 	{
-		uint256 _amt = getUint(getId, amt);
 
 		bool isEth = token == ethAddr;
 		address _token = isEth ? wethAddr : token;
 
 		IEulerDToken dToken = IEulerDToken(markets.underlyingToDToken(_token));
 
-		dToken.approveDebt(subAccountId, debtReceiver, _amt);
+		dToken.approveDebt(subAccountId, debtSender, amt);
 
-		setUint(setId, _amt);
+		setUint(setId, amt);
 
 		_eventName = "LogApproveDebt(uint256,address,address,uint256)";
```

Comment:
Should we update the event name as well `LogApproveSpenderDebt`?
```suggestion
		_eventName = "LogApproveDebt(uint256,address,address,uint256)";
```

Reply:
Can do! I've updated